src/extra/CoolComponent.spec.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
There are no tests in this file. You might want to look at `CoolComponent1.spec.js` which has the same component but with the corresponding test he wrote as well.